eng/Versions.props

```diff
@@ -2,7 +2,7 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.6.6</VersionPrefix>
+    <VersionPrefix>17.6.7</VersionPrefix>
```

Comment:
Leaving this now - it will get reset to 17.7.x later during the branding process